docs-kits/kits/Circularity_KIT/page_changelog.md

```diff
@@ -4,6 +4,10 @@ title: Changelog
 description: 'Circularity KIT'
 sidebar_position: 1
 ---
+## [0.2.0] - 2024-02-27
```

Comment:
@LuisRickert: Please change date to release data 2024-03-08

Reply:
Done